content/docs/1.7.0/snapshots-and-backups/scheduling-backups-and-snapshots.md

```diff
@@ -220,3 +222,11 @@ You can find the setting in Longhorn UI.
 When the setting is enabled, Longhorn will automatically attach the volume and take a snapshot/backup when it is time to do a recurring snapshot/backup.
 
 Note that during the time the volume was attached automatically, the volume is not ready for the workload. Workload will have to wait until the recurring job finishes.
+
+## Periodic Full Backup
+
+Longhorn performs delta backups by default, which means that only data that was changed since the last backup is uploaded. However, when a data block in the backupstore becomes corrupted, Longhorn does not replace that data block with a healthy one during subsequent backup operations. Corrupted data blocks in the backupstore may cause restoration operations to fail.
+When the `full-backup-interval` parameter is applied, Longhorn performs a full backup every N incremental backups. During a full backup, Longhorn uploads all data blocks in the volume. Data blocks that exist in the backupstore, including corrupted ones, are overwritten.
+
+> **Important**: 
+> Performing a full backup requires more time and incurs higher network throughput costs compared to the default incremental backup.
```

Comment:
Add `might` here. If the data in the volume are all overwritten, there is no difference between a full backup and an incremental backup.

```suggestion
> Performing a full backup might take longer and generate higher network throughput and costs than the default incremental backup.
```

Reply:
fixed.